ServerCore/Areas/Identity/Pages/Account/Edit.cshtml.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+
+namespace ServerCore.Areas.Identity.Pages.Account
+{
+    [AllowAnonymous]
```

Comment:
Can we only allow this for global admins? For players it would be good to always send them to the same place to edit their user settings

Reply:
OK, I'll restrict this.